src/riot-rs-embassy/src/lib.rs

```diff
@@ -15,7 +15,7 @@ pub type Task = fn(Spawner, TaskArgs);
 
 #[derive(Copy, Clone)]
 pub struct TaskArgs {
-    #[cfg(feature = "net")]
+    #[cfg(feature = "usb_ethernet")]
```

Comment:
Hm. The idea here is that network applications can just use "stack", independent of the underlying ethernet, wifi or 6lowpan.

Reply:
offline discussion: this is correct atm